exercises/practice/kindergarten-garden/kindergarten-garden_test.sql

```diff
@@ -0,0 +1,49 @@
+-- Setup test table and read in student solution:
+.read ./test_setup.sql
+
+-- Test cases:
+INSERT INTO tests (name, uuid, diagram, student, expected)
+    VALUES
+        ('garden with single student', '1fc316ed-17ab-4fba-88ef-3ae78296b692', 'RC
+GG', 'Alice', 'radishes,clover,grass,grass'),
```

Comment:
```suggestion
        ('garden with single student', '1fc316ed-17ab-4fba-88ef-3ae78296b692', 'RC' || char(10) || 'GG', 'Alice', 'radishes,clover,grass,grass'),
```

Thoughts?

Reply:
Nice. I like the explictness

Reply:
Looks good. Will fix later when I'm on my computer